fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/Auth.java

```diff
@@ -960,6 +960,12 @@ public void updateUserPropertyInternal(String user, List<Pair<String, String>> p
                 Env.getCurrentEnv().getEditLog().logUpdateUserProperty(propertyInfo);
             }
             LOG.info("finished to set properties for user: {}", user);
+        } catch (DdlException e) {
+            if (isReplay && e.getMessage().contains("Unknown user property")) {
```

Comment:
How about define a new Exception, not using `error message` to check?
And some some comment about how can it happen and why it can be ignored.

Reply:
done, thx